python/requirements/ml/data-test-requirements.txt

```diff
@@ -18,4 +18,5 @@ delta-sharing
 pytest-mock
 decord
 snowflake-connector-python
-pyiceberg[sql-sqlite]==0.7.0
\ No newline at end of file
+pyiceberg[sql-sqlite]==0.7.0
+hudi==0.2.0rc1
```

Comment:
Do we really need to peg to specific version?

Reply:
yes currently the api is not yet stabilized. and it's good to always freeze it to a version. i'll update this to `0.2.0` once release is out.